mercury.spec

```diff
@@ -14,7 +14,13 @@ License:	Aregonee National Laboratory, Department of Energy License
 URL:		http://mercury-hpc.github.io/documentation/
 Source0:	https://github.com/mercury-hpc/mercury/archive/v%{dl_version}.tar.gz
 
+%if 0%{?suse_version} > 0
+BuildRequires:	libatomic1
+%else
+%if 0%{?rhel} < 8
 BuildRequires:	openpa-devel
```

Comment:
Developers are locally building with libatomic on CentOS 7.  The scons is supposed to detect it.

If mercury still requires openpa, it means something else in DAOS  will fall back to openpa if libatomic is not present, and we need to get a ticket filed with Mercury upstream to remove the openpa dependency.

Reply:
They can't be.  I tried libatomic here and it fails to build.